### DISCUSSION TITLE
access denied for the repository to clone

### DISCUSSION THREAD

**Discussion by Maheshdvp999** [learndevopsonline/learndevopsonline#1281] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/134056076/c91ee375-f7bf-4d74-b649-a505f18cf363)
Access was denied when I want to clone a repository

**Comment by Naresh-vk-243** [comment]
i tried it and i can clone it, no issues found, please make sure you have access for that location where you are trying to clone in your local desktop
